src/packs/ranged/Shear Rifle.yml

```diff
@@ -1,16 +1,13 @@
 _id: YpribJStVIy5gPRG
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: true
     longReload: false
     max: 10
     suppress: false
     type: ammo
     value: 10
-  bundle:
-    amount: 1
-    bundled: false
   cost: 600
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.